packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -130,9 +130,19 @@ export const mutation = mutationType({
       args: {
         testingType: nonNull(arg({ type: TestingTypeEnum })),
       },
-      resolve: (source, args, ctx) => {
+      resolve: async (source, args, ctx) => {
         ctx.actions.project.setCurrentTestingType(args.testingType)
 
+        // if necessary init the wizard for configuration
+        if (ctx.coreData.currentTestingType
```

Comment:
@estrada9166 recently landed a PR that enables reconfiguring an existing project. It doesn't run though this mutation, so the `wizard.initialize` isn't called and no detection occurs. Maybe we move this up into `project.setCurrentTestingType` so that both scenarios are covered?